packages/plugin-auto-tracking-browser/README.md

```diff
@@ -0,0 +1,76 @@
+<p align="center">
+  <a href="https://amplitude.com" target="_blank" align="center">
+    <img src="https://static.amplitude.com/lightning/46c85bfd91905de8047f1ee65c7c93d6fa9ee6ea/static/media/amplitude-logo-with-text.4fb9e463.svg" width="280">
+  </a>
+  <br />
+</p>
+
+# @amplitude/plugin-auto-tracking-browser
+
+Official Browser SDK plugin for auto-tracking.
+
+## Installation
+
+This package is published on NPM registry and is available to be installed using npm and yarn.
+
+```sh
+# npm
+npm install @amplitude/plugin-auto-tracking-browser
+
+# yarn
+yarn add @amplitude/plugin-auto-tracking-browser
+```
+
+## Usage
+
+This plugin works on top of the Amplitude Browser SDK, generating auto-tracked events and sending to Amplitude.
+
+To use this plugin, you need to install `@amplitude/analytics-browser` version `v1.9.1` or later.
+
+### 1. Import Amplitude packages
+
+* `@amplitude/analytics-browser`
+* `@amplitude/plugin-auto-tracking-browser`
+
+```typescript
+import * as amplitude from '@amplitude/analytics-browser';
+import { autoTrackingPlugin } from '@amplitude/plugin-auto-tracking-browser';
+```
+
+### 2. Instantiate the plugin
+
+The plugin accepts 1 optional parameter, which is an `Object` to configure the allowed tracking options.
+
+```typescript
+const plugin = autoTrackingPlugin({
+  cssSelectorAllowlist: ['.amp-auto-tracking', '[amp-auto-tracking]'],
+  tagAllowlist: ['button', 'a'],
```

Comment:
do these need to be separate or can we unify?

Reply:
I thought about this. The function is a little bit overlapping but not all are easy to configure. For `tagAllowlist`, if it is not provided, we have a list of default tags that we want to track, while `cssSelectorAllowlist` is for more advanced usage, if doesn't have a default list. If we just rely on `cssSelectorAllowlist` with a default list like `['button', 'a']`, then problem is, when we want to add a allowed class, we have to do `['button.amp-auto-tracking', 'a.amp-auto-tracking']`, need to explicitly copy all tags here (this is what I think not easy to configure). Let me know if you have any good thoughts on this, this interface is not ready to public users at the moment though.